package.json

```diff
@@ -87,18 +85,30 @@
         "whatwg-fetch": "^3.0.0"
     },
     "dependencies": {
+        "@storybook/components": "^5.0.0",
+        "@storybook/core-events": "^5.0.0",
+        "@storybook/theming": "^5.0.0",
+        "preact": "^10.0.5",
+        "eslint": "^5.16.0",
```

Comment:
```
warning " > @typescript-eslint/eslint-plugin@1.13.0" has incorrect peer dependency "eslint@^5.0.0".
warning " > @typescript-eslint/parser@1.13.0" has incorrect peer dependency "eslint@^5.0.0".
```

In `incorrect peer dependencies` are addressed in this PR: https://github.com/libero/reviewer-client/pull/22

I opted to upgrade `@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser` rather than downgrade `eslint`.

This should remain a `devDependency`